x/oracle/types/ballot.go

```diff
@@ -84,12 +84,19 @@ func (pb ExchangeRateBallot) StandardDeviation() (sdk.Dec, error) {
 	}
 
 	sum := sdk.ZeroDec()
+	overflowCount := 0
 	for _, v := range pb {
 		deviation := v.ExchangeRate.Sub(median)
+		// check for overflow
+		err := checkMulOverflow(deviation, deviation)
+		if err != nil {
+			overflowCount++
+			continue
+		}
 		sum = sum.Add(deviation.Mul(deviation))
 	}
 
-	variance := sum.QuoInt64(int64(len(pb)))
+	variance := sum.QuoInt64(int64(len(pb) - overflowCount))
```

Comment:
Not a big fan of `overflowCount` being a separate variable, but this seemed like the cleanest way to account for skipped rates

Reply:
Never mind, realized a cleaner way